docs/index.html

```diff
@@ -1539,7 +1539,9 @@ <h3 id="oauth2">OAuth2</h3>
 <li>Zapier makes a backend call to your API to exchange the <code>code</code> for an <code>access_token</code>.</li>
 <li>Zapier stores the <code>access_token</code> and uses it to make calls on behalf of the user.</li>
 <li>(Optionally) Zapier can refresh the token if it expires.</li>
-</ol><p>You are required to define:</p><ul>
+</ol><blockquote>
+<p>Note: When <a href="https://platform.zapier.com/private_integrations/private-vs-public-integrations">building a public integration</a>,  the <code>redirect_uri</code> will change once the app is approved for publishing, to be more consistent with your app&#x2019;s branding. Depending on your API, you may need to add this new <code>redirect_uri</code> to an allow list in order for users to continue connecting to your app on Zapier. To access the new <code>redirect_uri</code>, run <code>zapier describe</code> again once the app is published.</p>
```

Comment:
Possibly related to https://github.com/zapier/zapier-platform/pull/661#issuecomment-1590131112 - I think the pre-commit hooks should have created these changes but for some reason they weren't